### DISCUSSION TITLE
Impossible to use QSV pro preview form

### DISCUSSION THREAD

**Discussion by aborruso** [dathere/qsv#1513] [General]
Hi @jqnatividad ,
I have tried to use it, but I have had this:

![image](https://github.com/jqnatividad/qsv/assets/30607/cb97cb3f-4160-406a-ba8c-e42783b4f28a)

Thank you

**Comment by jqnatividad** [comment]
Thanks @aborruso !

@samibaig , can you check why the contact form is not working?

**Reply by jqnatividad** [reply]
Hi @aborruso ,

It works on our side.

We do have some authentication logic on the form to reduce spam. Perhaps your browser is blocking the CloudFlare widget to ensure its a Human and not a bot filling up the form?

**Reply by aborruso** [reply]
That was probably it. Thank you, it worked
